plugins/org.jkiss.dbeaver.ext.db2/src/org/jkiss/dbeaver/ext/db2/model/DB2DataSource.java

```diff
@@ -75,47 +75,52 @@ public class DB2DataSource extends JDBCDataSource implements DBCQueryPlanner, IA
+    private final DBSObjectCache<DB2DataSource, DB2Tablespace> tablespaceCache = new JDBCObjectSimpleCache<>(DB2Tablespace.class, C_TS);
 
-    private final DB2GranteeCache                                groupCache         = new DB2GranteeCache(DB2AuthIDType.G);
-    private final DB2GranteeCache                                userCache          = new DB2GranteeCache(DB2AuthIDType.U);
+    private final DBSObjectCache<DB2DataSource, DB2RemoteServer> remoteServerCache
+        = new JDBCObjectSimpleCache<>(DB2RemoteServer.class, C_SV);
+    private final DBSObjectCache<DB2DataSource, DB2Wrapper> wrapperCache = new JDBCObjectSimpleCache<>(DB2Wrapper.class, C_WR);
+    private final DBSObjectCache<DB2DataSource, DB2UserMapping> userMappingCache = new JDBCObjectSimpleCache<>(DB2UserMapping.class, C_UM);
+
+    private final DB2GranteeCache groupCache = new DB2GranteeCache(DB2AuthIDType.G);
+    private final DB2GranteeCache userCache = new DB2GranteeCache(DB2AuthIDType.U);
 
     // Those are dependent of DB2 version
     // This is ok as they will never been called as the folder/menu is hidden in plugin.xml
-    private final DBSObjectCache<DB2DataSource, DB2StorageGroup> storagegroupCache  = new JDBCObjectSimpleCache<>(DB2StorageGroup.class, C_SG);
-    private final DBSObjectCache<DB2DataSource, DB2Role>         roleCache          = new JDBCObjectSimpleCache<>(DB2Role.class, C_RL);
-    private final DBSObjectCache<DB2DataSource, DB2Variable>     variableCache      = new JDBCObjectSimpleCache<>(DB2Variable.class, C_VR);
+    private final DBSObjectCache<DB2DataSource, DB2StorageGroup> storagegroupCache
+            = new JDBCObjectSimpleCache<>(DB2StorageGroup.class, C_SG);
+    private final DBSObjectCache<DB2DataSource, DB2Role> roleCache = new JDBCObjectSimpleCache<>(DB2Role.class, C_RL);
+    private final DBSObjectCache<DB2DataSource, DB2Variable> variableCache = new JDBCObjectSimpleCache<>(DB2Variable.class, C_VR);
+
 
-    private List<DB2Parameter>                                   listDBParameters;
-    private List<DB2Parameter>                                   listDBMParameters;
-    private List<DB2XMLString>                                   listXMLStrings;
+    private List<DB2Parameter> listDBParameters;
+    private List<DB2Parameter> listDBMParameters;
+    private List<DB2XMLString> listXMLStrings;
 
-    private DB2CurrentUserPrivileges                             db2CurrentUserPrivileges;
+    private DB2CurrentUserPrivileges db2CurrentUserPrivileges;
 
-    private String                                               schemaForExplainTables;
+    private String schemaForExplainTables;
 
-    private Double                                               version;                                                                                                                  // Database
+    private Double version;

```

Comment:
I don't think these changes were intentional

Reply:
I intentionally changed it.

Reply:
I found weird such formatting